server_tests/test/clientTests.js

```diff
@@ -843,194 +841,178 @@ describe("RUST API TEST SUITE", function() {
             return testClient.setOptions(sessionId, {'_bublVideoFileFormat': {'type':'mp4', 'width': 1920, 'height': 1920}})
-            .then( function onSuccess (res) {
-                Comparison.oscSetOptionsOutput(res);
-            }, wrapError);
+            .then( (res) => validate.done(res.body, schema.names.commandSetOptions),
+                wrapError);
         });
 
         it("Expect invalidParameterValue Error. camera.setOptions cannot set options when _bublVideoFileFormat option is set to unsupported value", function() {
             return testClient.setOptions(sessionId, {'_bublVideoFileFormat': 'UNSUPPORTED'})
             .then( expectError,
-                (err) => Comparison.invalidParameterValueError(err)
+                (err) => validate.error(err.error.response.body, schema.names.commandSetOptions, schema.errors.invalidParameterValue)
             );
         });
 
         it("Expect success. camera.setOptions successfully sets options when exposureDelay option is set to supported value", function() {
             return testClient.setOptions(sessionId, {'exposureDelay': 4})
-            .then( function onSuccess (res) {
-                Comparison.oscSetOptionsOutput(res);
-            }, wrapError);
+            .then( (res) => validate.done(res.body, schema.names.commandSetOptions),
+                wrapError);
         });
 
         it("Expect invalidParameterValue Error. camera.setOptions cannot set options when exposureDelay option is set to unsupported value", function() {
             return testClient.setOptions(sessionId, {'exposureDelay': -1})
             .then( expectError,
-                (err) => Comparison.invalidParameterValueError(err)
+                (err) => validate.error(err.error.response.body, schema.names.commandSetOptions, schema.errors.invalidParameterValue)
             );
         });
 
         it("Expect success. camera.setOptions successfully sets options when dateTimeZone option is set to supported value", function() {
             return testClient.setOptions(sessionId, {'dateTimeZone': '2015:07:23 14:27:39-04:00'})
-            .then( function onSuccess (res) {
-                Comparison.oscSetOptionsOutput(res);
-            }, wrapError);
+            .then( (res) => validate.done(res.body, schema.names.commandSetOptions),
+                wrapError);
         });
 
         it("Expect success. camera.setOptions successfully sets options when dateTimeZone option is set to supported value and bubl timezone", function() {
             return testClient.setOptions(sessionId, {'dateTimeZone': '2015:07:23 14:27:39-04:00|America/Toronto'})
-            .then( function onSuccess (res) {
-                Comparison.oscSetOptionsOutput(res);
-            }, wrapError);
+            .then( (res) => validate.done(res.body, schema.names.commandSetOptions),
```

Comment:
The `|Timezone` part is a Bublcam-specific extension. The other date-time format is valid and standard.